submodules-tasks.yml

```diff
@@ -0,0 +1,231 @@
+version: "3"
+vars:
+    G_DEP_DOWNLOAD_SCRIPT: "{{.ROOT_DIR}}/tools/scripts/download-dep.py"
+
+tasks:
+  default:
+    deps: ["clp-submodules", "core-submodules"]
+
+  clp-submodules:
```

Comment:
`log-viewer-submodules` (plural) sg

Reply:
On second thought, perhaps just `log-viewer` is sufficient since the tasks in this taskfile are essentially in the `submodules` namespace. Same with `core-submodules` -> `core`.